src/views/AnnualReport.vue

```diff
@@ -279,13 +367,24 @@
     </v-container>
 
     <!-- Buttons ( BEN/BC/CCC/ULC only ) -->
-    <v-container v-else-if="isBenBcCccUlc" id="bcorp-buttons-container" class="list-item">
-      <div class="buttons-left"></div>
+    <v-container
+      v-else-if="isBenBcCccUlc"
+      id="bcorp-buttons-container"
+      class="list-item"
+    >
+      <div class="buttons-left" />
 
       <div class="buttons-right">
-        <v-tooltip top color="#3b6cff" content-class="top-tooltip">
-          <template v-slot:activator="{ on }">
-            <div v-on="on" class="d-inline">
+        <v-tooltip
+          top
+          color="#3b6cff"
```

Comment:
I see hex colors in other views, too. Would be nice to fix all.

Reply:
If all else fails, we could declare a new color (tooltipColor?) in main.ts along with the others.

Reply:
So, I had a look where actual hex values are being used in the code. Turns out, only for the tooltip color. So, I declared a new color like you proposed Sev since I think that's the nicest way of going on about this.